src/RankedPairsCalculator.php

```diff
@@ -36,71 +41,55 @@ public function __construct(Ballot $tieBreakingBallot)
             $ballotTieBreaker = new BallotTieBreaker();
             $myTieBreakingBallot = $ballotTieBreaker->breakTiesRandomly($myTieBreakingBallot);
```

Comment:
This seems useful generally.  Just so you know, though, I've been doing this on the pivot side as well.  The idea is that if we do the random part on the pivot side, then the tideman library will act deterministically on the inputs.  That will allow checking against other tools.

Reply:
Sounds good. Currently the library lets the user decide which ballot to use as a tie breaker. If the user passes in a tie breaking ballot that contains ties, then the library breaks that ballot's ties randomly. If we need the library to produce deterministic results I think we would just need to ensure we were passing the RankedPairsCalculator constructor a totally ordered tie-breaking ballot. Does that make sense? Does that work for the use cases you have in mind?